posthog/batch_exports/service.py

```diff
@@ -258,6 +258,11 @@ def backfill_export(
 async def backfill_schedule(temporal: Client, schedule_id: str, schedule_backfill: ScheduleBackfill):
     """Async call the Temporal client to execute a backfill on the given schedule."""
     handle = temporal.get_schedule_handle(schedule_id)
+    description = await handle.describe()
+
+    if description.schedule.spec.jitter is not None:
+        schedule_backfill.end_at += description.schedule.spec.jitter
```

Comment:
Isn't the actual jitter a random value between 0 and `description.schedule.spec.jitter`? Do we not need to use the actual jitter applied per invocation?

(I could easily be wrong on both counts.)

Reply:
> Isn't the actual jitter a random value between 0 and description.schedule.spec.jitter?

Yes, the `jitter` attribute is more like "upper limit for jitter"

> Do we not need to use the actual jitter applied per invocation?

I don't think so: Temporal backfills work by figuring out how many runs would have ocurred in the backfill period if the Schedule had been running during that period. So, if we don't add anything to `end_at`, the last run of the period could have landed in `end_at + actual_jitter` which is outside the backfill period for any `actual_jitter > 0`.

By adding the "upper limit for jitter" to `end_at` this means that we are covering the full range where the last run could have landed. 

There could be issues depending on how we set the jitter: if we set jitter to larger than the period, we could be triggering duplicates by doing this. But our jitter is a flat 5 minutes for hourly workflows and 2 hours for daily exports, so this should be fine for now.

Reply:
We could use the actual jitter applied to each run, the problem is that we have no way of figuring out what Temporal is using for actual jitter, we only have the upper limit 🤔

Reply:
> By adding the "upper limit for jitter" to end_at this means that we are covering the full range where the last run could have landed.

Ahhh, I see. That makes more sense to me now.